### DISCUSSION TITLE
2.8.12 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#5258] [讨论]
· XAML 格式介绍中增加 FallbackSource 属性解释（LTCatt/PCL2Help#400，PR @shimoranla）
· 修复了以下 1 个 Bug
　　LTCatt/PCL2Help#397 导致打开帮助页面时报错（LTCatt/PCL2Help#399，PR @tangge233）
　　求求各位之后交 PR 前至少自己进去试一遍吧

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.8.12'>2.8.12</a>.</em>

**Comment by Joker2184** [comment]
6

**Comment by LinQingYuu** [comment]
>2.8.12

是的，正式版成 2.8.12 了（

**Reply by Joker2184** [reply]
欸wc 还得改.jpg）

没看到

更新日志的图片用错了
